src/disco/keyguard/fd_keyload.h

```diff
@@ -41,4 +41,22 @@ void FD_FN_SENSITIVE
 fd_keyload_unload( uchar const * key,
                    int           public_key_only );
 
+/* fd_keyload_alloc_protected_pages allocates `page_cnt` regular (4 kB)
+   pages of memory protected by `guard_page_cnt` pages of unreadable and
+   unwritable memory on each side.  Additionally the OS is configured so
+   that the page_cnt pages in the middle will not be paged out to disk
+   in a swap file, appear in core dumps, or be inherited by any child
+   process forked off from this process.  Terminates the calling process
```

Comment:
This comment (which I think I wrote) doesn't seem clear that the pages get wiped, not unmapped on fork